python_modules/libraries/dagster-looker/dagster_looker/api/components/looker_component.py

```diff
@@ -111,6 +121,23 @@ class LookerComponent(StateBackedComponent, Resolvable):
         ),
     ] = None
     translation: Optional[ResolvedMultilayerTranslationFn] = None
+    resource_key: str = "looker"
+    pdt_builds: Annotated[
+        Optional[list[PdtBuildConfig]],
+        Resolver.default(
+            model_field_type=PdtBuildConfig,
+            description="Configuration for starting PDT builds.",
+            examples=[
+                [
+                    {
+                        "model_name": "my_model",
+                        "view_name": "my_pdt_view",
+                        "force_rebuild": "true",
+                    }
+                ]
+            ],
+        ),
+    ] = None
```

Comment:
You should be able to make the core RequestStartPdtBuild object a subclass of `Resolvable`, which will make it possible to just make pdt_builds an `Annotated[Optional[list[RequestStartPdtBuild]], Resolver.default(description=..., examples=...)]`

(in this case, specifying the Resolver would just be done so that we can inject a description)

On that note, the description should be a bit more detailed, basically explaining that any request here will be converted into a materializable asset definition representing that PDT build.